pkg/universe.dagger.io/alpha/php/composer/image.cue

```diff
@@ -0,0 +1,37 @@
+package composer
+
+import (
+	"universe.dagger.io/docker"
+)
+
+// Go image default version
+_#DefaultVersion: "latest"
+
+// Build a go base image
+#Image: {
+	version: *_#DefaultVersion | string
+
+	packages: [pkgName=string]: version: string | *""
+
+	packages: git: _
+
+	docker.#Build & {
+		steps: [
+			docker.#Pull & {
+				source: "index.docker.io/composer:\(version)"
```

Comment:
As a follow-on, you could allow a custom image in addition to `version` like here: https://github.com/dagger/dagger/pull/2632/files  :)

Reply:
Thanks, updated my PR.